src/StreetNameRegistry/Municipality/LevenshteinDistanceCalculator.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace StreetNameRegistry.Municipality
+{
+    using System;
+
+    public class LevenshteinDistanceCalculator
```

Comment:
Can you add a comment where you found this? I'm assuming you didn't write it yourself :)

Reply:
chatgpt...

Reply:
Then `// Generated by ChatGPT`

Reply:
Have you also considered using https://www.nuget.org/packages/Fastenshtein ?
Seems like overkill for a package but less code for us to maintain